python/lancedb/table.py

```diff
@@ -149,13 +149,13 @@ class Table(ABC):
     @property
     @abstractmethod
     def schema(self) -> pa.Schema:
-        """The [Arrow Schema](https://arrow.apache.org/docs/python/api/datatypes.html#) of
-        this Table
+        """The [Arrow Schema](https://arrow.apache.org/docs/python/api/datatypes.html#)
+        of this Table
 
         """
         raise NotImplementedError
 
-    def to_pandas(self):
+    def to_pandas(self) -> pd.DataFrame:
```

Comment:
Have we done a round of clean up of pandas dependency, to make pandas optional. Can we check that the code still work if pandas is not installed.